src/components/Footer/index.js

```diff
@@ -1,32 +1,54 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import styled from 'styled-components'
 import { Padding } from 'styled-components-spacing'
+import { Location } from '@reach/router'
 
 import { Grid, Row, Col } from '../../components/grid'
 import { SectionTitle } from '../../components/Typography'
+import GetInTouch from '../Common/GetInTouch'
 import OfficeListing from './OfficeListing'
 import FooterLinks from './FooterLinks'
 
 export const GreyFooter = styled.footer`
   background: #232323;
 `
 
+const showGetInTouch = pathname => {
+  const path = pathname.replace(/^\/|\/$/gi, '')
+  return !['contact', 'training', 'speciality/node-js', 'join-us'].includes(
```

Comment:
I would have the array in `src/components/Header/navLinksHelper.js`

Reply:
I'd rather keep it in here. It's localised to the footer and doesn't need to be reused elsewhere

Reply:
No worries. If not used elsewhere it's valid to leave it here.